gradle-jdks/src/test/groovy/com/palantir/gradle/jdks/GradleJdkToolchainsIntegrationTest.groovy

```diff
@@ -175,6 +180,33 @@ class GradleJdkToolchainsIntegrationTest extends GradleJdkIntegrationSpec {
         gradleVersionNumber << GRADLE_TEST_VERSIONS
     }
 
+
+    def '#gradleVersionNumber: only generates daemon jdk'() {
+        gradleVersion = gradleVersionNumber
+        setupJdksHardcodedVersions('11')
```

Comment:
is this just setting up the directories? or does it do something else? it's hard to tell that this isn't the default behaviour if you didn't add `daemonJdkOnly` ?

Reply:
without `daemonJdkOnly` (default behaviour) we are generating directories for all jdks (see previous test: https://github.com/palantir/gradle-jdks/pull/472/files#diff-d8cb60820e935e74203f71da27d54ca1e390a3b5cb379b2a15f807309a3a4018R162).